ui/Screen/OrgRegistration.svelte

```diff
@@ -69,16 +70,19 @@
     // Start validating once the user enters something for the first time
     if (orgName && orgName.length > 0) validating = true;
     if (validating) validation.updateInput(orgName);
-
-    subject.name = orgName;
   }
 
   $: submitLabel =
     state === RegistrationFlowState.TransactionConfirmation
       ? "Submit transaction"
       : "Next";
-
   $: disableSubmit = $validation.status !== ValidationStatus.Success;
+
+  $: payer =
+    $session.status === Status.Success && getPayer($session.data.identity);
+  $: transaction = getTransaction(orgName);
+  $: subject = getSubject(orgName);
+  $: avatarFallback = generateAvatar(orgName);
```

Comment:
None of these need to be reactive, is that correct?

Reply:
Not necessarily; it seemed like a quick way to make them dependent on the `orgName` input value and also save some LOC later but I'm open to refactoring